### DISCUSSION TITLE
How to enable screen mirror for multi-screen?

### DISCUSSION THREAD

**Discussion by ChaojiangLuo** [WayfireWM/wayfire#1336] [Q&A] [answered]
I set the config file as:

=========================

 [output:eDP-1]
 mode = 1920x1080@60000
 position = 0,0
 transform = normal
 scale = 1.000000

 [output:HDMI-A-1]
 mode = mirror eDP-1
 layout = 0,0
 transform = normal
 scale = 1.000000

=========================

But the HDMI-A-1 screen is black,  why?

**Comment by ammen99** [comment, marked as answer]
The config format is correct. There is a bug in Wayfire which I had noticed as well, I have pushed a fix for it in this pull request: https://github.com/WayfireWM/wayfire/pull/1338

**Comment by ChaojiangLuo** [comment]
Good news. Thanks!
